Initium-Odp/war/odp/ajax_group.jsp

```diff
@@ -258,7 +258,10 @@
 								+ ")'>Demote from admin</a>");
 					}
 				}
-				if (groupStatus.equals("Kicked") == false)
+				if (groupStatus.equals("Kicked") == false 
+						&& groupStatus.equals("Admin") == false 
+						|| (Key) group.getProperty("creatorKey")).getId() == 
```

Comment:
Key equal tests should be done using GameUtils.equals()

Also please wrap your or logic in braces to indicate clearly what the intention is here.